plugins/woocommerce/includes/emails/class-wc-email-customer-pos-completed-order.php

```diff
@@ -228,9 +277,9 @@ public function add_unit_price( $item_id, $item, $order ) {
 		 * @return array Modified array of total rows.
 		 */
 		public function order_item_totals( $total_rows, $order, $tax_display ) {
-			$cash_payment_change_due_amount           = $order->get_meta( '_cash_change_amount', true );
-			$formatted_cash_payment_change_due_amount = wc_price( $cash_payment_change_due_amount, array( 'currency' => $order->get_currency() ) );
-			if ( ! empty( $cash_payment_change_due_amount ) ) {
+			$cash_payment_change_due_amount = $order->get_meta( '_cash_change_amount', true );
+			if ( '' !== $cash_payment_change_due_amount ) {
+				$formatted_cash_payment_change_due_amount     = wc_price( $cash_payment_change_due_amount, array( 'currency' => $order->get_currency() ) );
```

Comment:
![change in refund email](https://github.com/user-attachments/assets/9999bd5c-9102-4cc4-869b-882c7d593653)

I don't think we should display change in refund emails, 0 or otherwise.

When you get a refund, you just get your previous payment back... you don't get change. The figure we're using for change relates to the original purchase, but showing it here is a little confusing – is the shopper supposed to give change to the merchant? 😉

Reply:
Good observation & suggestion! I agree that it's confusing including this in the refund emails as the total amount changed. I removed this row for refund emails in https://github.com/woocommerce/woocommerce/pull/58012/commits/ae9001a162e6f85ddbb599cdb37486761b45711d.